specifyweb/frontend/js_src/lib/components/Notifications/Notifications.tsx

```diff
@@ -46,15 +46,20 @@ export function Notifications({
 
   function handleClearAll() {
     if (notifications === undefined) return;
-    notifications.forEach((notification) => {
-      ping('/notifications/delete/', {
-        method: 'POST',
-        // eslint-disable-next-line @typescript-eslint/naming-convention
-        body: formData({ message_id: notification.messageId }),
-        errorMode: 'dismissible',
-      });
+
+    // Map over the notifications array and get the messageId from each notification
```

Comment:
there is no need to comment each line line with detailed on what's happening
generally, leave comments when:
- you need to explain why some code is needed rather than what it is doing
- when what it is doing is complicated/no obvious from looking at it (though in those cases, better naming the variables and refactoring code into smaller functions often removes the need for the comments)